lib/experimental/fonts/font-library/class-wp-rest-font-collections-controller.php

```diff
@@ -86,7 +86,7 @@ public function get_items( $request ) {
 		if ( $page > $max_pages && $total_items > 0 ) {
 			return new WP_Error(
 				'rest_post_invalid_page_number',
-				__( 'The page number requested is larger than the number of pages available.', 'default' ),
+				__( 'The page number requested is larger than the number of pages available.', 'gutenberg' ),
 				array( 'status' => 400 )
```

Comment:
The use of `default` here is actually intentional.

This string is directly from Core, so my understanding is that we can specify the `default` text domain here to use the existing Core translation and not re-translate it in the Gutenberg plugin. (But please correct me if it doesn't work that way!)

Reply:
In my opinion, all text domains in the Gutenberg plugin should be `gutenberg`. Otherwise, if this string is changed in the core, we will not be able to translate it unless you change the string in the Gutenberg plugin as well.

However, if this string has been in the core for several years and is stable, `default` text domain might be fine.

@swissspidy @oandregal I remember discussing a similar issue in #53888. Which domain do you think we should use?